src/components/overlays/IDPDelete/index.tsx

```diff
@@ -70,7 +70,13 @@ const getIntro = (status: Status, retry: () => void) => {
     return (
       <Trans i18nKey="[span]overlays.idp_delete_error_intro">
         Please{' '}
-        <span className="idp-retry-link" onClick={retry}>
+        <span
+          className="idp-retry-link"
+          onClick={retry}
+          onKeyUp={() => {
+            //doChange
```

Comment:
this function doesn't to anything - please change this (and all the other ones in this PR) to the comment we have used in those cases before

`// do nothing`

Reply:
@oyo This is solution wrt this ticket https://jira.catena-x.net/browse/CPLP-3488
I found this as a solution. What you think ?

Reply:
@nidhigarg-bmw I think it doesn't really matter what comment we write but it should be understandable to the reviewer. `doChange` is somehow irritating when actually nothing happens. The eslint example for this rule was `// do nothing` which seems more appropriate to me and also we used this for empty functions in the past. That's why I'd like to stick with that comment.